lldb/test/API/lang/swift/archetype_in_expression/TestArchetypeInExpression.py

```diff
@@ -0,0 +1,37 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import unittest2
+
+
+class TestBoundOpaqueArchetype(TestBase):
+
+    @swiftTest
+    def test(self):
+        """Tests that a user can refer to the archetypes in their expressions"""         
+        self.build()
+
+        # Check that you can refer to all types and function's archetypes.
+        _, process, _, breakpoint = lldbutil.run_to_source_breakpoint(self, 
+                "break here", lldb.SBFileSpec("main.swift"))
+        self.expect("e First.self", substrs=["Int.Type"])
```

Comment:
what happens if you try to `frame variable T`? Is it an error? Might be nice to document that with a test.

Reply:
`frame variable T` will fail (currently). I'll document this in the test